### DISCUSSION TITLE
Is there support for the Logitech G923?

### DISCUSSION THREAD

**Discussion by Aarnav-Sharma** [AntiMicroX/antimicrox#896] [Q&A]
Doesn't show up but AFAIK people have used AntiMicro/X with the G29

**Comment by BoomerangAide** [comment]
Saw this: "If you quit G HUB before connecting the G923 then it will be in Xinput mode" (instead of Direct Input).

**Reply by Aarnav-Sharma** [reply]
I see. It should already be in XInput mode then given that G HUB is not installed on my PC. I tried disconnecting and reconnecting via USB but still no detection whatsoever.
